apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/s3/SharedS3Utilities.java

```diff
@@ -97,7 +96,9 @@ public static S3AsyncClient createS3AsyncClient(Region awsS3Region) {
     if (minioConfig.useMinio) {
       return createS3AsyncMinioClient(awsS3Region, minioConfig);
     }
-    return S3CrtAsyncClient.builder().region(awsS3Region).build();
+    // Split the file download (automatically using the async client) if its >200 MB
+    long mb = 1048576L;
+    return S3AsyncClient.crtBuilder().minimumPartSizeInBytes(200 * mb).region(awsS3Region).build();
```

Comment:
This seems reasonable. What's the default part size?

I don't think there's any harm in doing this, but it doesn't seem like we're _too_ confident in this change being the solution. I'm building the images based off this PR now and I can deploy the result to `prod-sbx` so we can evaluate it **before** acceptance. What do you think?

Reply:
Yup that seems like a good idea. The default previously was 8MB, so a 400MB file would be split into 50 parts. I'm still not entirely sure what the issue is given that things work fine locally, so a test on the box would be prudent.